Auth0.Windows/Auth0Client.cs

```diff
@@ -199,10 +205,20 @@ public Task<JObject> GetDelegationToken(string targetClientId, IDictionary<strin
         /// <summary>
         /// Log a user out of a Auth0 application.
         /// </summary>
-        public void Logout()
+        /// <param name="forceClearCache">Force clearing the browser cache, no matter the login method</param>
+        public void Logout(bool forceClearCache = false)

```

Comment:
Thing is, I'm still not sure what the ClearCache() Method does exactly? Thus I figured that for some reason, someone might for example NOT want to clear the cache, even though the BrowserAuthenticationForm has been used. Using my proposed code, by default in this situation, the cache would be cleared. Having a second overload for Logout allows the consumer of the library to NOT clear the cache. If this is a scenario which is not likely to happen or doesn't make any sense, then by all means let's make it the simplest possible and use your proposed snippet.